README.md

```diff
@@ -0,0 +1,249 @@
+ - [Credentials](docs/Credentials.md)
+ - [CredentialsData](docs/CredentialsData.md)
+ - [CredentialsDataRoles](docs/CredentialsDataRoles.md)
+ - [CredentialsDataRolesBusinessDetails](docs/CredentialsDataRolesBusinessDetails.md)
+ - [CredentialsDataRolesBusinessDetailsLogo](docs/CredentialsDataRolesBusinessDetailsLogo.md)
+ - [Details](docs/Details.md)
+ - [DetailsData](docs/DetailsData.md)
+ - [DetailsDataEndpoints](docs/DetailsDataEndpoints.md)
+ - [EnvironmentalImpactCategoryType](docs/EnvironmentalImpactCategoryType.md)
+ - [Evse](docs/Evse.md)
+ - [EvseStatusSchedule](docs/EvseStatusSchedule.md)
+ - [GenericError](docs/GenericError.md)
+ - [GenericResponse](docs/GenericResponse.md)
+ - [Image](docs/Image.md)
+ - [InterfaceRoleType](docs/InterfaceRoleType.md)
+ - [LocationReferences](docs/LocationReferences.md)
+ - [Locations](docs/Locations.md)
+ - [LocationsData](docs/LocationsData.md)
+ - [LocationsDataOpeningTimes](docs/LocationsDataOpeningTimes.md)
+ - [LocationsDataOpeningTimesExceptionalOpenings](docs/LocationsDataOpeningTimesExceptionalOpenings.md)
+ - [LocationsDataOpeningTimesRegularHours](docs/LocationsDataOpeningTimesRegularHours.md)
+ - [LocationsDataPublishAllowedTo](docs/LocationsDataPublishAllowedTo.md)
+ - [LocationsDataRelatedLocations](docs/LocationsDataRelatedLocations.md)
+ - [ModuleIDType](docs/ModuleIDType.md)
+ - [OcpiChargingprofilePostRequest](docs/OcpiChargingprofilePostRequest.md)
+ - [OcpiCommandsCommandPostRequest](docs/OcpiCommandsCommandPostRequest.md)
+ - [OcpiLocationsLocationIdGet200Response](docs/OcpiLocationsLocationIdGet200Response.md)
+ - [Price](docs/Price.md)
+ - [ReservationRestrictionType](docs/ReservationRestrictionType.md)
+ - [ReserveNow](docs/ReserveNow.md)
+ - [Session](docs/Session.md)
+ - [SessionChargingPeriods](docs/SessionChargingPeriods.md)
+ - [SessionChargingPeriodsDimensions](docs/SessionChargingPeriodsDimensions.md)
+ - [SessionResponse](docs/SessionResponse.md)
+ - [SessionTotalCosts](docs/SessionTotalCosts.md)
+ - [SessionsResponse](docs/SessionsResponse.md)
+ - [SetChargingProfile](docs/SetChargingProfile.md)
+ - [StartSession](docs/StartSession.md)
+ - [StopSession](docs/StopSession.md)
+ - [Tariff](docs/Tariff.md)
+ - [TariffDeleteResponse](docs/TariffDeleteResponse.md)
+ - [TariffElements](docs/TariffElements.md)
+ - [TariffElementsRestrictions](docs/TariffElementsRestrictions.md)
+ - [TariffEnergyMix](docs/TariffEnergyMix.md)
+ - [TariffEnergyMixEnvironImpact](docs/TariffEnergyMixEnvironImpact.md)
+ - [TariffResponse](docs/TariffResponse.md)
+ - [TariffsResponse](docs/TariffsResponse.md)
+ - [Token](docs/Token.md)
+ - [TokenEnergyContract](docs/TokenEnergyContract.md)
+ - [TokenResponse](docs/TokenResponse.md)
+ - [TokensResponse](docs/TokensResponse.md)
+ - [UnlockConnector](docs/UnlockConnector.md)
+ - [Versions](docs/Versions.md)
+ - [VersionsData](docs/VersionsData.md)
+
+
+## Documentation For Authorization
+
+Endpoints do not require authorization.
```

Comment:
This is not entirely true - most of the modules require Bearer Token authorization per OCPI.

Reply:
I removed it, it was autogenerated